downstream/assemblies/platform/assembly-using-rhsso-operator-with-automation-hub.adoc

```diff
@@ -0,0 +1,37 @@
+ifdef::context[:parent-context: {context}]
+
+[id="using-rhsso-operator-with-automation-hub"]
+
+:context: using-a-rhsso-operator
+
+= Using a Red Hat Single Sign On Operator with {HubName}
+
+The _{PlatformName}_ (AAP) _{PrivateHubNameStart}_ (PAH) uses Red Hat Single Sign-On (RH-SSO) for external authentication.
+
+The Red Hat Single Sign-On Operator automates Red Hat Single Sign-On administration in Openshift.
+Use this Operator to create custom resources, which automate administrative tasks.
+
+When installing AAP on VMs the installer can automatically install and configure RH-SSO for use with PAH.
```

Comment:
VMs = virtual machine. Spell out the first occurrence of an acronym even if it's commonly known. Same for PAH.

Also what is RH-SSO? That isn't a product name.